Fritz.Chatbot/Commands/AttentionCommand.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using Fritz.StreamLib.Core;
+using Microsoft.Extensions.Configuration;
+
+
+namespace Fritz.Chatbot.Commands
+{
+  public class AttentionCommand : IBasicCommand
+  {
+	private readonly IConfiguration Configuration;
+
+	public AttentionCommand(IConfiguration configuration)
+	{
+	  this.Configuration = configuration;
+	}
+
+	public string Trigger => "attention";
+
+	public string Description => "Play audio queue to divert attention to chat";
+
+	public TimeSpan? Cooldown => TimeSpan.FromSeconds(5);
+
+	public async Task Execute(IChatService chatService, string userName, ReadOnlyMemory<char> rhs)
+	{
+	  var player = new NetCoreAudio.Player();
+	  await player.Play("hey_listen.wav");
```

Comment:
The chatbot will not be running on the streamer's machine... we will need a way to route this to the host

Reply:
I believe a solution for this would be to use SignalR. You would create a webpart page that plays the audio via an HTML5 Audio control from javascript whenever the SignalR method is raised. Jeff could then add the webpart to OBS as another browser source.